dips/storage_service_upload.py

```diff
@@ -110,32 +109,22 @@ def main(
         "events": [],
         "agents": [],
     }
-    # SS 'file/async/' POST request.
     # TODO: Move this to amclient.
-    url = "%s/api/v2/file/async/" % ss_url
+    LOGGER.info("Storing DIP in SS.")
+    url = "%s/api/v2/file/" % ss_url
     headers = {"Authorization": "ApiKey %s:%s" % (ss_user, ss_api_key)}
-    response = requests.post(url, headers=headers, json=dip_data, timeout=5)
+    response = requests.post(
+        url, headers=headers, json=dip_data, timeout=86400
```

Comment:
I think it's okay like that as it's only used in here for now.

Wherever you see this kind of timeout in web requests, it's a workaround to a bigger problem. You shouldn't wait that much for a response and long running tasks should be deferred to the background to avoid that, that's what we were trying to fix with the async. endpoint.